components/scream/tests/scream_tests.cpp

```diff
@@ -0,0 +1,23 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch2/catch.hpp"
+#include "coupler/coupler.hpp"
+#include "p3/p3.hpp"
+#include "rrtmgp/rrtmgp.hpp"
+#include "shoc/shoc.hpp"
+
+namespace {
```

Comment:
Is there a real need for the empty namespace? Just curious, not that it hurts...

Reply:
We don't want to export any unit testing symbols. This has been common practice on some teams I've been on in the past.

Reply:
But this is a single translation unit, right? It won't be included anywhere, so the symbols would say confined to the file.

Anyhow, it doesn't hurt, so I'm fine with it. I was just curious.

Reply:
@bartgol , i think it also protects against link errors

Reply:
They are still exported. A function in an anonymous namespace is extern-equivalent to a static function.

Reply:
Sounds good. Thanks!